editors/code/src/commands/index.ts

```diff
@@ -13,5 +14,6 @@ export {
     matchingBrace,
     parentModule,
     runnables,
-    syntaxTree
+    syntaxTree,
+    on_enter,
```

Comment:
I guess *prettier* and *tslint* are going to complain... But if they don't, you should probably use `onEnter` and remove the `,`